examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>if you set DEVITO_LOGGING to PERF the time elapsed in the operator is printed.</p><p>You can set via:</p><p>configuration['logging']='perf'...</p><p>so you can get rid of the timings above....</p><p> </p><p>Ahhhh sorry....</p><p>you get that Operator <code>Kernel</code> run in 0.02 s so  no need for the rest.</p><p> </p><p>Keep only <code>op(time=nt, dt=dt0)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
I put this command configuration['DEVITO_LOGGING'] = 'PERF' but a error appeared!

Reply:
What is the error?

Reply:
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-2-224e9c1ec1e2> in <module>
      3 from   examples.seismic  import Receiver
      4 from   devito            import SubDomain, Grid, NODE, TimeFunction, Function, Eq, solve, Operator, configuration
----> 5 configuration['DEVITO_LOGGING'] = 'PERF'

~/anaconda3/envs/devito/devito/parameters.py in wrapper(self, key, value)
     60                         % (key, self._deprecated[key]))
     61                 key = self._deprecated[key]
---> 62             return func(self, key, value)
     63         return wrapper
     64 

~/anaconda3/envs/devito/devito/parameters.py in wrapper(self, key, value)
     45     def _check_key_value(func):
     46         def wrapper(self, key, value):
---> 47             accepted = self._accepted[key]
     48             if accepted is not None:
     49                 tocheck = list(value) if isinstance(value, dict) else [value]

KeyError: 'DEVITO_LOGGING'

Reply:
Done!